pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -625,6 +625,7 @@ self: super:
       if (!isDarwin)
       then {
         outputs = [ "out" "dev" ];
+        patches = [ ./dont-create-logdir.patch ];
```

Comment:
What confuses me is that this change, and where all the defaults are changed are set in different commits (the last one, and the commit after this one). If you could squash them all into once, and connect everything in the commit message, it'd be great.

Reply:
The xorgserver patches don't actually change the defaults (or any behaviour in NixOS for that matter). They are patches which make workarounds we had to add in NixOS obsolete.